src/app/views/river-detail/__tests__/river-detail.spec.js

```diff
@@ -3,7 +3,7 @@ test.todo('revive tests')
 // import { createWrapper } from '@/utils'
```

Comment:
now or sometime else could we just delete this file out. There's not really a point to maintaining a test that doesn't work anymore and the mapbox->maplibre isn't really adding anything to this file..